pyproject.toml

```diff
@@ -67,7 +67,7 @@ test = [
 
 [project.urls]
 Changelog = "https://glass.readthedocs.io/stable/manual/releases.html"
-Documentation = "https://glass.readthedocs.io/"
+Documentation = "https://glass.readthedocs.io/stable"
```

Comment:
Again I think this just needs to point to the base URL.

Reply:
https://github.com/glass-dev/glass/pull/416#discussion_r1843681121

Reply:
What about this one (cc: https://github.com/glass-dev/glass/pull/416#discussion_r1843720045)? The `Changelog` has to have `stable` in it by RTD design

Reply:
@ntessore think its just this comment left

Reply:
Indeed, makes sense to have it at stable!